cmd/tracee-rules/readme.md

```diff
@@ -1,6 +1,5 @@
 # Documentation
 
-The full documentation of Tracee-Rules is available at
-[https://aquasecurity.github.io/tracee/dev/docs/detecting/](https://aquasecurity.github.io/tracee/dev/docs/detecting/).
-You can use the version selector on top to view documentation for a specific
-version of Tracee.
+**Important**
+Tracee has no a new design.
```

Comment:
```suggestion
Tracee has a new design.
```
???

Reply:
ohh did I misunderstand? I thought this is related to the previous division that we had in Tracee?